src/element/Hyperlink.tsx

```diff
@@ -72,246 +72,8 @@ export const Hyperlink = ({
     toast: { message: string; closable?: boolean; duration?: number } | null,
   ) => void;
 }) => {
-  const appState = useExcalidrawAppState();
```

Comment:
Add a early return on the **top**, and don't remove the code. We want as little changes as possible